tokio/src/util/slab.rs

```diff
@@ -157,6 +157,10 @@ struct Slot<T> {
 
     /// Next entry in the free list.
     next: u32,
+
+    /// Makes miri happy. Could probably also be fixed by replacing `slots` with a raw-pointer
```

Comment:
it would be nice if this comment explained *why* this makes miri happy (i'm guessing it's related to mutable-noalias, but IDK...)